src/products/components/ProductVariants/utils.tsx

```diff
@@ -199,32 +321,42 @@ export function getColumnData(
     };
   }
 
-  if (isStockColumn.test(name)) {
+  if (getColumnStock(name)) {
     return {
       ...common,
       width: 100,
-      title: warehouses.find(
-        warehouse => warehouse.id === name.match(isStockColumn)[1],
-      )?.name,
+      title: warehouses.find(warehouse => warehouse.id === getColumnStock(name))
+        ?.name,
+    };
+  }
+
+  if (getColumnChannel(name)) {
+    const channel = channels.find(
+      channel => channel.id === getColumnChannel(name),
+    );
+    return {
+      ...common,
+      width: 150,
+      title: `Price in ${channel.name} [${channel.currency}]`,
+    };
+  }
+
+  if (getColumnChannelAvailability(name)) {
+    const channel = channels.find(
+      channel => channel.id === getColumnChannelAvailability(name),
+    );
+    return {
+      ...common,
+      width: 80,
+      title: `Available in ${channel.name}`,
```

Comment:
Same here

Reply:
This too, left it here before i get to styling